ddtrace/contrib/aiohttp/__init__.py

```diff
@@ -1,7 +1,36 @@
 """
-The ``aiohttp`` integration traces all requests defined in the application handlers.
+The ``aiohttp`` integration traces requests made with the client or to the server.
 
-Automatic instrumentation is not available for ``aiohttp.web.Application``, instead
+
+Enabling
+~~~~~~~~
+
+Client
+******
+
+The client integration is enabled automatically when using
+:ref:`ddtrace-run<ddtracerun>` or :func:`patch_all()<ddtrace.patch_all>`.
+
+Or use :func:`patch()<ddtrace.patch>` to manually enable the integration::
+
+    from ddtrace import patch
+    patch(aiohttp=True)
+
+
+Global Configuration
+~~~~~~~~~~~~~~~~~~~~
+
+.. py:data:: ddtrace.config.aiohttp['distributed_tracing']
```

Comment:
do we need "client" specific naming for config?

```python
ddtrace.config.aiohttp_client["distributed_tracing"]
```

?

since we do have a server component as well

Reply:
Yeah since the majority of users are probably using the client and the `aiohttp` entry already exists I opted to stick with the name `aiohttp` but I do like being more explicit. Let's opt for `aiohttp_client` 👍